src/units/attack_type.hpp

```diff
@@ -104,7 +104,8 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	 * @param special_id If true, match @a special against the @c id of special tags.
 	 * @param special_tags If true, match @a special against the tag name of special tags.
 	 */
-	bool has_special(const std::string& special, bool simple_check=false, bool special_id=true, bool special_tags=true) const;
+	bool has_special(const std::string& special, bool special_id=true, bool special_tags=true) const;
+	bool has_special_simple(const std::string& special, bool special_id=true, bool special_tags=true) const;
```

Comment:
Needs a Doxygen comment. What exactly makes this simpler from the other `has_special`? Maybe there's a better name for it.

Reply:
I the name comes form the simple_check parameter which was there before, but basicially has_special returns actve specials, and simple returns speciasl of theis attack

Reply:
So it's simple because it doesn't check if the special is active?